### PR TITLE
x3 all timeouts

### PR DESCRIPTION
**Pull request by kelson42**
In an attempt to make mwoffliner less sensitive to slow network/backends

### CODE REVIEW

File: lib/mwoffliner.lib.js [hunk 1]
```diff
@@ -220,8 +220,36 @@ function execute(argv) {
                     parsoidConfig.setMwApi({
                         uri: '${mw.base + mw.apiPath}',
                     });
+
+                    parsoidConfig.timeouts = {
+
+                        // How long does a request have to generate a response?
+			request: 12 * 60 * 1000,
```
Comment (subbuss) [inline lib/mwoffliner.lib.js]: 12 mins seems very long! On the WMF cluster we use 2 mins, but in reality, p99 numbers are ~60 sec. But, not harmful to use 12 mins .. just noticing. Maybe start with 5 mins? In any case, I want to extract this into a separate config file as well and eventually add options to pass in other config files.

**Comment by subbuss** [comment]
Overall, since scraping is not time-sensitive, increasing timeouts to these large values is not a problem per se. So, I am okay with it.

File: lib/mwoffliner.lib.js (hunk 1 above)
Comment (kelson42) [reply lib/mwoffliner.lib.js]: Yes... I just try to find where things go wrong with #149 ... and in fact this PR seems to no 100% fix the problem. See my last comment there.

**Comment by kelson42** [comment]
OK, so I run this on a stronger machine and it works... something here is not fully under control, but this is good enought to me for now.

**Pull request closed without merging**